service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/LegalRepresentativesUpdatedHandler.java

```diff
@@ -31,12 +37,19 @@ public void sendEmailToLegalRepresentativesAddedToCase(final LegalRepresentative
             unwrapElements(caseData.getLegalRepresentatives())
         );
 
-        legalRepresentativesChange.getAdded().forEach(
-            legalRepresentative -> notificationService.sendEmail(
-                LEGAL_REPRESENTATIVE_ADDED_TO_CASE_TEMPLATE,
+        Set<LegalRepresentative> added = legalRepresentativesChange.getAdded();
+
+        if (!added.isEmpty()) {
```

Comment:
do you need the isEmpty() check?

Reply:
technically no but this was me being very picky on whether I make the request to LD or not